shared/bash_remediation_functions/fix_audit_syscall_rule.sh

```diff
@@ -81,6 +81,7 @@ then
 elif [ "$tool" == 'augenrules' ]
 then
 	# Extract audit $key from audit rule so we can use it later
+	declare -a matches=()
```

Comment:
What was the error here? Normally, it is OK to just write `matches=()`.

Reply:
`covscan` thinks that there is no assignment for `matches`. But I guess we can use just `matches=()` then.

Reply:
So let's do the same with `privileged_binaries`.

Reply:
Done